media/examples/rocket.svg

```diff
@@ -0,0 +1,15 @@
+<svg width="28" height="15" xmlns="http://www.w3.org/2000/svg" xmlns:svg="http://www.w3.org/2000/svg">
+ <g>
+  <g transform="rotate(90 15,10)">
+   <g>
+    <g>
+     <g>
+      <path d="m9.89987,-0.86855c0.67152,-0.97175 1.39264,-1.97525 2.41907,-2.5974c1.01214,0.60672 1.72619,1.59238 2.38812,2.5469c1.55915,2.36168 2.51239,5.19836 2.3201,8.04938c0.50386,0.42236 1.0223,0.82787 1.51866,1.25888c0.71253,0.6549 1.04523,1.67649 0.87368,2.62621c-0.23904,1.19973 -0.46879,2.40217 -0.72178,3.59862c-0.14065,0.61555 -0.97346,0.89737 -1.46308,0.5041c-0.81268,-0.67835 -1.60515,-1.38167 -2.41759,-2.06088c-0.6147,0.59154 -1.43128,0.98599 -2.28858,1.0335c-0.98689,0.06959 -1.96395,-0.33967 -2.67942,-1.0083c-0.5829,0.44624 -1.12336,0.9819 -1.69287,1.45985c-0.27526,0.2212 -0.50415,0.51189 -0.82991,0.66293c-0.50505,0.21669 -1.15544,-0.10671 -1.2696,-0.64959c-0.26007,-1.16968 -0.53177,-2.33728 -0.78416,-3.50877c-0.20511,-0.99102 0.15816,-2.07004 0.92176,-2.73444c0.44206,-0.37129 0.89029,-0.73512 1.34061,-1.0963c0.12546,-0.06598 0.06242,-0.21816 0.06954,-0.32994c-0.13424,-2.75077 0.80537,-5.46973 2.29546,-7.75475zm0.79759,3.62326c-0.58973,0.75919 -0.51839,1.93514 0.16557,2.61316c0.72411,0.79787 2.09567,0.82697 2.85306,0.06123c0.58465,-0.53415 0.78235,-1.43094 0.50329,-2.16787c-0.25571,-0.71044 -0.94028,-1.2434 -1.6932,-1.30967c-0.6953,-0.08003 -1.41523,0.23985 -1.82872,0.80314z" fill="#020202"/>
+     </g>
+     <path d="m9.61844,15.53727c-0.01011,-0.3449 0.41913,-0.58204 0.70925,-0.39953c1.22797,0.63226 2.75437,0.63198 3.9823,-0.00033c0.28181,-0.17302 0.69948,0.03479 0.70987,0.37039c0.00413,0.73251 0.00498,1.46578 0.00114,2.19853c0.00565,0.35757 -0.4667,0.5856 -0.74433,0.36198c-0.21132,-0.18636 -0.4019,-0.39592 -0.60521,-0.59149c-0.32164,0.62661 -0.62481,1.26272 -0.9491,1.88786c-0.15574,0.29786 -0.64237,0.2999 -0.8011,0.00498c-0.3261,-0.62633 -0.62718,-1.26623 -0.95509,-1.89194c-0.20036,0.19732 -0.39151,0.40575 -0.60255,0.59182c-0.27645,0.22471 -0.74789,-0.00769 -0.74405,-0.36298c-0.00589,-0.72288 -0.00204,-1.44609 -0.00114,-2.16929z" fill="#020202"/>
+    </g>
+    <circle r="0.98788" cy="3.98804" cx="12.30356" fill="#020202"/>
+   </g>
+  </g>
+ </g>
+</svg>
```

Comment:
Nit: missing empty line at end of file. I would also suggest running this through https://jakearchibald.github.io/svgomg/

Reply:
Thanks! I didn't know about this tool.